EventsExpress.Db/DbInitialize/DbInitializer.cs

```diff
@@ -62,6 +62,10 @@ public static class DbInitializer
                 new Category{ Name="Team-Building"},
                 new Category{ Name="Swimming"},
                 new Category{ Name="Gaming"},
+                new Category {Name = "Fishing"},
```

Comment:
It would be better to use the same code stile